### PR TITLE
Hide Labs menu item in Tchap 

### PR DESCRIPTION
**Pull request by ClaireGizard**
#380

### CODE REVIEW

File: vector/src/main/res/xml/vector_settings_root.xml [hunk 1]
```diff
@@ -42,7 +42,8 @@
     <im.vector.app.core.preference.VectorPreference
         android:icon="@drawable/ic_settings_root_labs"
         android:title="@string/room_settings_labs_pref_title"
-        app:fragment="im.vector.app.features.settings.VectorSettingsLabsFragment" />
+        app:fragment="im.vector.app.features.settings.VectorSettingsLabsFragment"
+        app:isPreferenceVisible="@bool/labs_menu_item_visible"/>
```
Comment (Florian14) [inline vector/src/main/res/xml/vector_settings_root.xml:46]: please reformat, a space is missing before the closing tag

File: vector/build.gradle [hunk 2]
```diff
@@ -327,6 +327,9 @@ android {
 
             applicationId "fr.gouv.rie.tchap"
             versionName "${versionMajor}.${versionMinor}.${versionPatch}${getTchapVersionSuffix()}" + "_b"
+
+            // Tchap: Show labs menu item only in btchap
+            resValue "bool", "labs_menu_item_visible", "true"
```
Comment (Florian14) [inline vector/build.gradle:332]: You will probably say that I'm quibbling, but could you rename it to something like "show_labs_settings"? It is not really a menu item but a whole setting category. :)

File: vector/build.gradle [hunk 3]
```diff
@@ -327,6 +327,9 @@ android {
 
             applicationId "fr.gouv.rie.tchap"
             versionName "${versionMajor}.${versionMinor}.${versionPatch}${getTchapVersionSuffix()}" + "_b"
+
+            // Tchap: Show labs menu item only in btchap
```
Comment (Florian14) [inline vector/build.gradle:331]: I think that you can remove the `only` word

File: vector/build.gradle [hunk 4]
```diff
@@ -336,6 +339,9 @@ android {
 
             applicationId "fr.gouv.tchap.a"
             versionName "${versionMajor}.${versionMinor}.${versionPatch}${getTchapVersionSuffix()}"
+
+            // Tchap: Hide labs menu item only in tchap
```
Comment (Florian14) [inline vector/build.gradle:343]: Same here

File: vector/build.gradle (hunk 2 above)
Comment (ClaireGizard) [reply vector/build.gradle:332]: i used the same format like for developper mode :/

Comment (ClaireGizard) [reply vector/build.gradle:332]: labs_settings_visible ?

**Review by giomfo — APPROVED** [review]

**Review by Florian14 — APPROVED** [review]
LGTM

**Pull request merged**